Ihor/Netflix/src/HomePage/SearchPanel/SearchPanel.tsx

```diff
@@ -38,38 +39,29 @@ class SearchPanel extends React.Component<
       search: string;
     };
     let { searchBy, search } = query;
-    if (searchBy) {
-      if (this.state.searchBy === IsearchBy.title) {
-        this.setState({ searchBy: IsearchBy.title });
-      } else {
-        this.setState({ searchBy: IsearchBy.genre });
-      }
-    } else {
-      this.setState({ searchBy: IsearchBy.title });
+    if(searchBy){
```

Comment:
almost correct. please, try one more.. 

ps: you don't need to wrap in searchBy. If there be a "search", you won't set it, because searchBy is empty.